### ISSUE TITLE
Create syntatic sugar method similar to getBalance

### ISSUE THREAD

**Issue by vladyslavstartsev** [iexbase/tron-api#31]
it would be nice to have similar method to `getBalance` that will get balance of an asset,
I think the signature should look like this
`getTokenBalance(string $tokenID, string $address = null, bool $formated = false): float`

the internal code should look like, 

```php
   public function getTokenBalanceByAddress(string $tokenId, string $address = null, bool $formatted = false): float
    {
        $account = $this->tron->getAccount($address);

        if(!array_key_exists('assetV2', $account)) {
            return 0.0;
        }

        $amount = 0.0;
        foreach ($account['assetV2'] as $asset) {
            if ($asset['key'] === $tokenId) {
                $amount = $asset['value'];
            }
        }
        if ($amount === 0.0) {
            return $amount;
        }

        if ($formatted) {
            $tokenInfo = $this->tron->getTokenFromID($tokenId);
            if (array_key_exists('precision', $tokenInfo)) {
                $precision = $tokenInfo['precision'];
            } else {
                $precision = 0;
            }
            return (float)bcdiv((string)$amount, bcpow('10', (string)$precision), $precision);
        }
        return $amount;
    }
```

**Comment by serderovsh** [comment]
You can implement this function in your project in parallel.
